chemprop/v2/tests/unit/data/test_dataset.py

```diff
@@ -70,8 +70,8 @@ def test_none():
 
 
 def test_empty():
-    """TODO"""
-    pass
+    with pytest.raises(ValueError):
+        MoleculeDataset([], MoleculeMolGraphFeaturizer())
 
```

Comment:
why should an empty dataset be invalid?

Reply:
What is the `test_empty` test intends to test? My initial thought is that it expects error when the data is empty.